feel/.gitignore

```diff
@@ -0,0 +1,15 @@
+/.bundle/
+/doc/
+/docs/boneyard/
+*.log
+/pkg/
+/tmp/
+.byebug_history
+
+.DS_Store
+node_modules
+
+Gemfile.lock
+package-lock.json
+yarn.lock
+coverage
```

Comment:
Might as well move this to the root with some minor path modifications so we don't have to repeat it.